frontend/src/components/StocksYouFollow/StocksYouFollow.jsx

```diff
@@ -39,30 +45,78 @@ export default function StocksYouFollow({ stocks, setStocks }) {
         fetchStocksYouFollow();
     }, []);
 
-    return (
+    return stocks.length > 0 ? (
         <div className="stocks-you-follow">
             <h1>Stocks You Follow</h1>
-            <div className="stocks">
-                {stocks?.map((stock) => {
-                    return (
-                        <div key={stock.ticker} className="stock-margin">
-                            <Link
-                                key={stock.name}
-                                to={`/search/stocks/${stock.ticker}`}
-                                className="stock-link"
-                            >
-                                <div className="stock">
-                                    <img
-                                        src={stock.logo}
-                                        alt={`This is a logo of ${stock.name}.`}
-                                    />
-                                    <p>{stock.ticker}</p>
-                                </div>
-                            </Link>
-                        </div>
-                    );
-                })}
-            </div>
+            <Table
+                className="rounded-table"
+                striped
+                bordered
+                hover
+                variant="dark"
+            >
+                <thead>
+                    <tr>
+                        <th className="text-center">Logo</th>
+                        <th className="text-center">Ticker</th>
+                        <th className="text-center">Price</th>
+                        <th className="text-center">Sector</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {stocks
+                        .slice(0, Math.min(displayedStocks, stocks.length))
+                        .map((stock) => {
+                            return (
+                                <tr key={stock.ticker}>
+                                    <td className="stock-image">
+                                        <div className="stock-follow-logo">
+                                            <img
+                                                src={stock.logo}
+                                                alt={`This is a logo of ${stock.name}.`}
+                                                onLoad={() =>
```

Comment:
Inline is a good idea if the callback is short. I'd recommend just defining an event handler for `onLoad`. 

Is the reason you did this inline so that you can reuse the `stock` parameter, since this `img` is inside a `map`? This will also be simpler if you create a child component for each Row, so the `onLoad` handler can read in the props (see example in [React docs](https://react.dev/learn/responding-to-events#reading-props-in-event-handlers))